components/range.js

```diff
@@ -0,0 +1,62 @@
+const template = `
+<div class="c-range" :class="classes">
+  <label class="c-range__label" :for="name">
+    <span dir="ltr">{{label}}</span>
```

Comment:
For code style purposes, let's keep a space between the curlybraces and the content.